### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by havogt**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,54 +1,39 @@
...
+FetchContent_Declare(
+  c_bindings
+  GIT_REPOSITORY https://github.com/GridTools/c_bindings.git
+  GIT_TAG        master # consider replacing master by a tagged version
+)
+FetchContent_GetProperties(c_bindings)
+if(NOT c_bindings_POPULATED)
+  FetchContent_Populate(c_bindings)
+  add_subdirectory(${c_bindings_SOURCE_DIR} ${c_bindings_BINARY_DIR})
+endif()
```
Comment (lukasm91) [inline README.md:24]: should we mention FetchContent_MakeAvailable again? In my opinion that should be the preferred way as long as no old CMake versions need to be supported.

File: README.md [hunk 2]
```diff
@@ -1,54 +1,39 @@
...
-- Intel is able to compile GridTools code, but depending on user code, might have severe performance problems compared to GNU- or Clang-compiled code.
-
-##### Officially not supported (no workarounds implemented and planned)
-
-| Compiler | Backend | Date | Comments
-| --- | --- | --- | --- |
-| NVCC <= 9.1 with GNU 6.x | cuda | 2018-10-16 | similar to [this tuple bug](https://devtalk.nvidia.com/default/topic/1028112/cuda-setup-and-installation/nvcc-bug-related-to-gcc-6-lt-tuple-gt-header-/)
-| PGI 18.5 | x86 | 2018-12-06 | no effort to fix compilation
-| Cray 8.7.3 | x86 | 2018-12-06 | no effort to fix compilation
+- The library doesn't work with NVCC <= 9.2 using Clang as host compiler, if c++14 is enabled, due to a bug in NVCC.
```
Comment (lukasm91) [inline README.md:36]: Did you verify again that the bug occurs only <=9.2?

**Review by lukasm91 — APPROVED** [review]
two remarks

File: README.md (hunk 2 above)
Comment (havogt) [reply README.md:36]: yes

File: README.md (hunk 1 above)
Comment (havogt) [reply README.md:24]: Let's keep it for now. I think anyway it make sense to update our CMake at some point, then we can also fix it here.

**Pull request merged**